db/migrate/20220828142008_remove_drupal_gallery_from_node.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveDrupalGalleryFromNode < ActiveRecord::Migration[5.2]
+  def change
+    Node.all.each do |node|
+      new_revision = node.revisions.first.dup
+      gallery_images = ""
+      node.gallery.each do |image|
```

Comment:
OK, I changed this and directly referenced it as `node&.drupal_content_field_image_gallery.each do |image|`

https://github.com/publiclab/plots2/commit/4e80ae2cf6f30aad9515f324ca5ceecce3697c17

Reply:
And noting the fixes @anirudhprabhakaran3 had added in this PR too: https://github.com/publiclab/plots2/pull/11393